apps/docs/app/icons/page.tsx

```diff
@@ -0,0 +1,18 @@
+"use client";
+
+import { allPages } from "@/.contentlayer/generated";
+import { Mdx } from "@/components/Mdx/MdxComponent";
+
+export default function Page() {
+    return <>
+        {allPages
+            .filter(page => page._id === "pages/icons.mdx")
```

Comment:
filter and map usually mean there is multiple element matching this. It doesn't make sense to see a <main> inside a loop, since there should only be 1 main in a html document. 

you should use find if you want to find a single page
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find